docs/src/tutorial/simple_system.md

```diff
@@ -162,11 +224,16 @@ If you are not sure whether you have the latest version, please upgrade to ensur
 
 -   In the *Object parameter* table (typically at the top-center),
     select the *demand* parameter and the *Base*
-    alternative, and enter the value *100* as seen in the
-    image below. This will establish that there's a demand of '100'
+    alternative, and enter the value *150* as seen in the
+    image below. This will establish that there's a demand of '150'
     at the electricity node.
 ![image](../figs_simple_system/simple_system_electricity_demand.png)
 
+!!! note
+    The alternative name is not optional. If you don't select *Base* 
+    (or another name) you will not be able to save your data.
+    Speaking of which, when is the last time you saved/committed?
```

Comment:
This is funny, but maybe also useful to remind them _how_ to save/commit, since it wasn't super clear the first time that that's what you should be doing regularly. It just says "Go here - commit - comment that you're saving." Especially for users who aren't familiar with what "commit" means.

Reply:
I've made it more explicit the first time the commit instruction is given.